packages/edit-site/src/components/header-edit-mode/index.js

```diff
@@ -43,6 +46,12 @@ function Header( { setEntitiesSavedStatesCallback } ) {
 					: undefined
 			}
 		>
+			<Button
+				className="edit-site-layout__view-mode-toggle"
+				onClick={ () => setCanvasMode( 'view' ) }
+			>
+				<SiteIcon className="edit-site-layout__view-mode-toggle-icon" />
+			</Button>
```

Comment:
The other good thing here is that this makes the header identical to the post editor and unifying this button is now possible.